openmetadata-ui/src/main/resources/ui/src/rest/glossaryAPI.ts

```diff
@@ -166,13 +158,10 @@ export const importGlossaryInCSVFormat = async (
   data: string,
   dryRun = true
 ) => {
-  const configOptions = {
-    headers: { 'Content-type': 'text/plain' },
-  };
   const response = await APIClient.put<string, AxiosResponse<CSVImportResult>>(
     `/glossaries/name/${getEncodedFqn(glossaryName)}/import?dryRun=${dryRun}`,
     data,
-    configOptions
+    APPLICATION_JSON_CONTENT_TYPE_HEADER
```

Comment:
It will change the content type from text/plain to JSON is it intentional?

Reply:
No, I will revert it, thanks for catching it.